### DISCUSSION TITLE
Start the server after shutting it down with the Shutdown command

### DISCUSSION THREAD

**Discussion by SOSkr** [thijsvanloef/palworld-server-docker#179] [Q&A] [answered]
What would be the best way to start the server after having shut it down with the rcon command?

Right now I do it by restarting the container, but is there another way?

**Comment by thijsvanloef** [comment, marked as answer]
@SOSkr The best way would be to simply stop the container if you want to shutdown the server, the server saves and shuts down gracefully when RCON is enabled. 
And starting the container when you want to resume the server

**Reply by SOSkr** [reply]
I had some issues using the stop command, additionally I want to send a message before sending the shutdown command.

thats why I use rcon.
